build.xml

```diff
@@ -28,29 +28,34 @@
                 <exclude name="**/phpspec/**" />
                 <exclude name="**/phpunit/**" />
                 <exclude name="**/sebastian/**" />
-                <exclude name="**/symfony/**" />
-            </fileset>					
+                <exclude name="**/symfony/yaml/**" />
```

Comment:
Seems like vendor/symfony/polyfill-ctype/bootstrap.php gets added to the composer autoloader, which is used by the phar, so this needs to be here.